Add Linux install guide for Ubuntu 20.04 LTS

ratzrattillo opened the pull request:
Add Linux install guide for Ubuntu 20.04 LTS
Update Windows install guide
Include fontawesome-free packages in package.json

ratzrattillo committed:
Add Linux install guide for Ubuntu 20.04 LTS

Update Windows install guide
Include fontawesome-free packages in package.json

bastibl commented:
Thanks for the PR! Is there a reason why you added fontawesome? It looks like it's never used. Or did I miss something?

And one more question regarding Windows, since I don't have much experience here: Did you need SDRuno to interface an RTL-SDR? Isn't the driver part of Pothos SDR?

ratzrattillo commented:
@bastibl Fontawesome is used for the icons that are included, like the small github or discord symbols.

Without SDRUno i could not open any device based on SoapySDR. This is really weird, but i believe its the latest SoapySDR library shipping with SDRUno...

without SDRUno and just with PhotosSDR i get:

`
[INFO] [UHD] Win32; Microsoft Visual C++ version 14.2; Boost_107500; UHD_4.0.0.HEAD-0-g90ce6062
Found Rafael Micro R820T/2 tuner
FutureSDR: ERROR - SoapySDR::Device::enumerate(sdrplay) sdrplay_api_Open() failed
FutureSDR: INFO - Using format CF32.
`

bastibl commented:
But you are just adding fontawesome to the node modules, no? It's not included in the build, i.e., it doesn't end up in the dist folder. And there are no fontawesome icons created. Or do I miss something obvious here?

Any chance you used a device filter with the soapy source? Quite some time back I used the RTL-SDR and HackRF on Windows without SDRuno, but might have changed since then.

ratzrattillo committed:
Fix unneeded dependency

ratzrattillo commented:
Oh yes, you are right. i removed again the fontawesome dependency, that i added previously. And i also adjusted the Readme again. I had issues with the dependencies on my system.

I tried to open the device with and without a device filter. Both led to the same issue.
Using your hint, that you pasted in the chat on 21.03 did the trick:
https://discourse.myriadrf.org/t/pothos-win10-sdrplay-api-open-error-sdrplay-api-fail/7331

bastibl commented:
Ah, I see. Thank you! It's online :-)
